catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -104,9 +108,18 @@ public UpdateRequest process(UpdateRequest input)
                 validator.validate(metacard);
             } catch (ValidationException e) {
                 String validatorName = getValidatorName(validator);
+                boolean validationErrorsExist = CollectionUtils.isNotEmpty(e.getErrors());
+                boolean validationWarningsExist = CollectionUtils.isNotEmpty(e.getWarnings());
 
-                if (isValidatorEnforced(validatorName)) {
-                    INGEST_LOGGER.info(
+                if (isValidatorEnforced(validatorName) && validationErrorsExist && enforceErrors) {
+                    INGEST_LOGGER.debug(
```

Comment:
These conditions could be cleaned up and collapsed to reduce duplication, the bodies of the conditions are identical.